src/agentscope/_init.py

```diff
@@ -85,6 +87,9 @@ def init(
     dir_log = str(file_manager.dir_log) if save_log else None
     setup_logger(dir_log, logger_level)
 
+    # Set monitor
+    _ = MonitorFactory.get_monitor(db_path=file_manager.file_db)
```

Comment:
Where to set the total budget? It's suggested to add arguments in `init` function